src/_listObject.scss

```diff
@@ -0,0 +1,220 @@
+
+    .#{$block}__object {
+      width: 100%;
+
+      &--top-row {
+        display: flex;
+        vertical-align: $fd-list-object-with-vertical-align-top;
+      }
+
+      &--top-left {
+        width: 60%;
+
+        @include fd-rtl() {
+          text-align: $fd-list-object-align-right;
+        }
+      }
+
+      &--top-right {
+        width: 40%;
+        text-align: $fd-list-object-align-right;
+        padding-left: $fd-list-object-with-padding-value-one !important;
+        vertical-align: $fd-list-object-with-vertical-align-top !important;
+        margin-right: $fd-list-object-with-value-zero !important;
+
+        @include fd-rtl() {
+          text-align: $fd-list-object-align-left;
+          padding-right: $fd-list-object-with-padding-value-one !important;
+          padding-left: $fd-list-object-with-value-zero !important;
+          margin-left: $fd-list-object-with-value-zero !important;
+        }
+      }
+
+      &--bottom-row {
+        display: flex;
+        align-items: flex-end;
+        justify-content: flex-end;
+
+        > div:only-child {
+          padding: $fd-list-object-with-value-zero;
+        }
+      }
+
+      &--bottom-left {
+        text-align: $fd-list-object-align-left;
+        padding:
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-value-zero;
+
+        @include fd-rtl() {
+          text-align: $fd-list-object-align-right;
+          padding:
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-value-zero
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-padding-value-half;
+        }
+      }
+
+      &--bottom-right {
+        @include fd-ellipsis();
+
+        width: 50%;
+        text-align: $fd-list-object-align-right;
+        margin-right: $fd-list-object-with-value-zero;
+        padding:
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-value-zero
+          $fd-list-object-with-padding-value-half
+          $fd-list-object-with-padding-value-half;
+
+        @include fd-rtl() {
+          text-align: $fd-list-object-align-left;
+          margin-left: $fd-list-object-with-value-zero;
+          padding:
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-padding-value-half
+            $fd-list-object-with-value-zero;
+        }
+      }
+
+      &--bottom {
+        margin-top: 0.25rem;
+      }
+    }
+
+    .#{$block}__object-number {
+      vertical-align: $fd-list-object-with-vertical-align-top !important;
+      margin-right: $fd-list-object-with-value-zero;
+
+      &-text {
+        font-weight: normal;
+        font-size: 1.375rem;
+        font-family: $fd-list-object-number-font-family;
+        text-shadow: $fd-list-object-with-no-text-shadow;
```

Comment:
Same as unit - the only thing that should stay in this block is the font-size.

Reply:
text-shawdow -added to stay in object list item look as per Fiori3 specs.(as mentioned in the above comments.)